examples/config-builder/resonator.py

```diff
@@ -0,0 +1,48 @@
+import numpy as np
+
+from qualang_tools.config.configuration import *
+from qualang_tools.config.components import *
+from qualang_tools.config.builder import ConfigBuilder
+
+cont = Controller("con1")
+
+res = ReadoutResonator(
+    "res1",
+    outputs=[cont.analog_output(0), cont.analog_output(1)],
+    inputs=[cont.analog_input(0), cont.analog_input(1)],
+    intermediate_frequency=2e6,
+)
+res.lo_frequency = 4e9
+
+wfs = [
+    ArbitraryWaveform("wf1", np.linspace(0, -0.5, 16).tolist()),
+    ArbitraryWaveform("wf2", np.linspace(0, -0.5, 16).tolist()),
+]
+
+ro_pulse = MeasurePulse("ro_pulse", wfs, 16)
+ro_pulse.add(
+    Weights(ConstantIntegrationWeights("integ_w1_I", cosine=1, sine=0, duration=16))
+)
+ro_pulse.add(
+    Weights(ConstantIntegrationWeights("integ_w1_Q", cosine=0, sine=-1, duration=16))
+)
+ro_pulse.add(
+    Weights(ConstantIntegrationWeights("integ_w2_I", cosine=0, sine=1, duration=16))
+)
+ro_pulse.add(
+    Weights(ConstantIntegrationWeights("integ_w2_Q", cosine=1, sine=0, duration=16))
+)
+
+res.add(Operation(ro_pulse))
+
```

Comment:
We need a few comments.
At this part, I forgot that the "cont" is the controller...

At the beginning, write something like:
First we would define the controller, elements and operations

Then here write something like:
Now that we have everything ready, we can create the configuration. First we create a ConfigBuilder object, and then we add the controller and elements to it.

Reply:
is there any documentation available for `py-qua-tools` that I can refer here?

Reply:
There is nothing for py-qua-tools in general. Baking has this:
https://github.com/qua-platform/py-qua-tools/tree/main/qualang_tools/bakery
The only other "big" tool is the config builder...

Reply:
I added few comments, but we need a detailed guide on "how to use ConfigBuilder?" in `py-qua-tools`

Reply:
I agree...

Reply:
there will be a new PR in py-qua-tools